server/server.js

```diff
@@ -14,6 +14,9 @@ app.use(express.static(path.join(__dirname, 'static')))
 
 app.use('/posts', routes)
 
+app.get('*', (request, response) => {
+  response.sendFile(path.resolve(__dirname, '../static', 'index.html'))
+})
```

Comment:
This isn't the correct way to set up a static route in Express. Use the static middleware instead. I believe this is already in place in `server/index.js`.